code/world.dm

```diff
@@ -8,3 +8,4 @@
 	view = "15x15"
 	cache_lifespan = 0	//stops player uploaded stuff from being kept in the rsc past the current session
 	fps = 20 // If this isnt hard-defined, anything relying on this variable before world load will cry a lot
+	name = "Paradise Station 13"
```

Comment:
Required so dmdoc reads the correct project name instead of just `Space Station 13`.